apps/docs/components/components/counter.md

```diff
@@ -68,34 +81,45 @@ The root element of `SfCounter` is a `<span>`. Since a `<span>` is an inline ele
 :::
 
 ::: slot api
+
 ## Props
 
-| Prop name | Type              | Default value | Possible values                       |
-|-----------|-------------------|---------------|---------------------------------------|
-| `size`      | `SfCounterSize` | `'base'`        | `'3xs'`, `'2xs'`, `'xs'`, `'sm'`, `'base'`, `'lg'` |
-| `pill`      | `boolean`         | `false`       |                                       |
+| Prop name | Type            | Default value | Possible values                                    |
+| --------- | --------------- | ------------- | -------------------------------------------------- |
```

Comment:
question: wondering, are you doing this formatting of table manually or via some extension ?

Reply:
extension, prettier maybe?